addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/CsrfCountermeasuresScanRule.java

```diff
@@ -136,9 +131,18 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
             for (Element formElement : formElements) {
                 LOGGER.debug(
                         "FORM [{}] has parent [{}]", formElement, formElement.getParentElement());
+                ++numberOfFormsPassed;
```

Comment:
Maybe rename this `formId` and move the init to just above this loop? I think thats what its used for now, and the current name confused me..